pkg/process/events/model/model_common.go

```diff
@@ -4,12 +4,16 @@
 // Copyright 2016-present Datadog, Inc.
 
 //go:generate go run github.com/tinylib/msgp -tests=false
+//go:generate go run github.com/DataDog/datadog-agent/pkg/security/generators/event_copy -scope "(p *ProcessConsumer)" -os linux -pkg consumer -output ../consumer/event_copy_linux.go ProcessEvent .
+//go:generate go run github.com/DataDog/datadog-agent/pkg/security/generators/event_copy -scope "(p *ProcessConsumer)" -os windows -pkg consumer -output ../consumer/event_copy_windows.go ProcessEvent .
 
 //nolint:revive // TODO(PROC) Fix revive linter
 package model
 
 import (
 	"time"
+
+	smodel "github.com/DataDog/datadog-agent/pkg/security/secl/model"
```

Comment:
I "think" this is the reason of the import explosion

Reply:
Yeah, I knew when I did this that it would cause issue :( , I'll used what I introduce (cast) in the copy generator to remove this dependency

Reply:
Should be fixed